dialog/file.go

```diff
@@ -42,9 +42,12 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 				f.open.Enable()
 			}
 		}
+		saveName.Wrapping = fyne.TextTruncate
```

Comment:
We should not truncate an entry, the user needs to be able to edit the whole thing. A ScrollContainer is probably the right solution.

Reply:
Ack